apis/kueue/v1beta1/workload_types.go

```diff
@@ -59,6 +59,15 @@ type WorkloadSpec struct {
 	// +kubebuilder:default=""
 	// +kubebuilder:validation:Enum=kueue.x-k8s.io/workloadpriorityclass;scheduling.k8s.io/priorityclass;""
 	PriorityClassSource string `json:"priorityClassSource,omitempty"`
+
+	// QueueingPolicy that will determine if a job needs to be requeued or not after being suspended
+	// Possible values are:
+	//
+	// - Never: indicates that a suspended job should not be queued
+	//  it will not suspend the job automatically
+	// - Always: indicates that a suspended job should be queued
+	//   if there's quota available the job should resume automatically
```

Comment:
```suggestion
	// - Never: indicates that a suspended job should not be queued
	//   it will not suspend the job automatically.
	// - Always: indicates that a suspended job should be queued
	//   if there's quota available the job should resume automatically.
	// +kubebuilder:validation:Enum={Never, Always}
	// +kubebuilder:default="Always"
```